frontend/src/routes/FacilityBooking/ViewConflicts/ConflictBookingModal.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react'
+import { ConfirmationModal } from '../../../components/ConfirmationModal'
+import { handleCreateBooking } from '../../../store/facilityBooking/action'
+import { Booking } from '../../../store/facilityBooking/types'
+import { useHistory } from 'react-router-dom'
+import { PATHS } from '../../Routes'
+
+export default function ConflictBookingModal(props: { booking: Booking | undefined }) {
```

Comment:
might want to write this exactly? since there is only 1 prop
```suggestion
export default function ConflictBookingModal({booking}: { booking: Booking | undefined }) {
```

Reply:
Resolved in commit f24c30e8b814ee8bc691b9803db23a6fc53234a6.